modules/ROOT/pages/storage.adoc

```diff
@@ -4,6 +4,8 @@ Fedora CoreOS ships with a simple default storage layout: the root partition is
 
 Below, we provide examples of various ways you can customize this.
 
+NOTE: Fedora CoreOS requires the root filesystem to be at least 5G. If adding additional partitions after the root filesystem, you must make sure to explicitly resize the root partition as shown below. If the root filesystem is less than 5G, a warning is emitted on login. In the future, this will trigger a hard provisioning failure by default. For more details, see https://github.com/coreos/fedora-coreos-tracker/issues/586.
```

Comment:
Hmm.  This says what the user should do but doesn't really explain why.  I tried adding more words, but perhaps I've just turned this into a wall-o-text instead.  WDYT?

```suggestion
[NOTE]
====
Fedora CoreOS requires the root filesystem to be at least 5 GiB. The default root filesystem is smaller than 5 GiB on some platforms, but is automatically resized to fill all available disk space. If you add additional partitions after the root filesystem, you must make sure to explicitly resize the root partition as shown below.

Currently, if the root filesystem is smaller than 5 GiB, a warning is emitted on login. In the future, if the root filesystem is smaller than 5 GiB, Fedora CoreOS will refuse to boot. For more details, see https://github.com/coreos/fedora-coreos-tracker/issues/586[this bug].
====
```

Reply:
Thanks, I took your text and just massaged it a bit more.